README.md

```diff
@@ -3,7 +3,10 @@
 package_updater
 ===========
 
-Basic description for package_updater
+Run the ansible package module to install, upgrade or remove
```

Comment:
I don't know that we need to specifically say how we do it. IMO, the description should be more of a "what" and less of a "how".

Reply:
README.md redone